streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -48,11 +48,11 @@
 import org.apache.kafka.streams.errors.UnknownStateStoreException;
 import org.apache.kafka.streams.errors.InvalidStateStorePartitionException;
 import org.apache.kafka.streams.internals.metrics.ClientMetrics;
-import org.apache.kafka.streams.processor.Processor;
```

Comment:
One super frustrating thing in this PR was that you _can't_ suppress deprecation warnings on imports, so I have had to use the fully qualified class name of the deprecated APIs _everywhere_.

Aside from a lot of noise in the PR (sorry about that), this means that we have some pretty long lines. It's not always easy to get these lines under the limit without making them even harder to read due to weird line breaks. I've followed my best judgement about when it's better to just keep a line long.